manifests/forwarder.pp

```diff
@@ -182,7 +182,7 @@
   Optional[String[1]] $package_provider      = $splunk::params::package_provider,
   Boolean $manage_package_source             = true,
   Optional[String[1]] $package_source        = undef,
-  Array[String[1]] $install_options          = $splunk::params::forwarder_install_options,
+  $install_options                           = $splunk::params::forwarder_install_options, # Not strongly typed because can be Hash or Array
```

Comment:
Can we not use `Variant[Hash, Array[String[1]]]`?

Reply:
This is the error I get when I try that:
parameter 'install_options' variant 0 expects a Hash value, got Tuple
parameter 'install_options' variant 1 index 9 expects a String value, got Struct

Reply:
Trying with `Variant[Tuple[Hash,String,1,default], Array[String[1]]]` for a reverse-order of the $install_options

Reply:
Verified that this works.